### PR TITLE
add emlid to the list of GPS devices supported

### PR DESCRIPTION
**Pull request by barchstien**
@dagar @mhkabir

### CODE REVIEW

**Comment by barchstien** [comment]
@hamishwillee

**Comment by hamishwillee** [comment]
Thanks @barchstien ! 

Is this an RTK GPS as indicated by the link? IN that case, there needs to be an update to https://docs.px4.io/en/gps_compass/rtk_gps.html page and probably remove the section you added here: " GPS Options"

Thoughts?

**Comment by barchstien** [comment]
> Thanks @barchstien !
> 
> Is this an RTK GPS as indicated by the link? IN that case, there needs to be an update to https://docs.px4.io/en/gps_compass/rtk_gps.html page and probably remove the section you added here: " GPS Options"
> 
> Thoughts?

Hi @hamishwillee 
RTK is not supported yet. I planned to start on it next week
The reason for the "GPS Options" section is that emlid reach M+ does not contain a compass
The way I use it on pixhawk 4, is together with the default GPS+Compass still connected (but GPS unused) and emlid reach plugged to the uart/i2c port (GPS_1_CONFIG set to 104)

File: en/gps_compass/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # GPS & Compass
 
-PX4 supports global navigation satellite systems (GNSS) (including GPS, GLONASS, Galileo, BeiDou, QZSS and SBAS) using receivers that communicate via the UBlox, MTK or Ashtech protocols, or via UAVCAN. It also supports [Real Time Kinematic (RTK) GPS Receivers](../gps_compass/rtk_gps.md), which extend GPS systems to centimetre-level precision.
+PX4 supports global navigation satellite systems (GNSS) (including GPS, GLONASS, Galileo, BeiDou, QZSS and SBAS) using receivers that communicate via the UBlox, MTK Ashtech or Emlid protocols, or via UAVCAN. It also supports [Real Time Kinematic (RTK) GPS Receivers](../gps_compass/rtk_gps.md), which extend GPS systems to centimetre-level precision.
```
Comment (mhkabir) [inline en/gps_compass/README.md]: missing comma after MTK

Comment (barchstien) [reply en/gps_compass/README.md]: ooops my bad, just fixed it

**Comment by hamishwillee** [comment]
@barchstien - Thanks for your response https://github.com/PX4/px4_user_guide/pull/432#issuecomment-460952941

OK then probably worth adding a note in that section that RTK GPS not yet supported for this item. When it is supported, the change I suggested should be made (doc in RTK GPS section and remove in this section)

**Comment by barchstien** [comment]
@hamishwillee I added a comment on RTK not being supported at the moment

**Comment by hamishwillee** [comment]
@barchstien Thanks for this. I couldn't find your RTK clarification in this PR, so I have merged and then added my own - https://github.com/PX4/px4_user_guide/commit/29e90ee5f9c1c65705883be90ca65bd280bb5803

Can you please just sanity check that note?

**Comment by barchstien** [comment]
@hamishwillee Looks fine
Thanks mate

**Comment by barchstien** [comment]
Hi Ahmad

I have no feedback for PX4, and no way to test it, sorry about that
My team moved from PX4 to Ardupilot, mostly for in flight behaviour which
was more according to our need with Ardupilot

Normally, you just got to setup something like mavlink-router, that can
inject RTCM data as Mavlink packet to the link
So no more dev is required on the Firmware, only GCS config/dev depending
on your topology

Cheers
Bastien

**Comment by Ahmad-Drak** [comment]
Thanks for the clarification @barchstien

**Pull request merged**